solidity/test/lib/deploy.ts

```diff
@@ -0,0 +1,53 @@
+// set this to turn off paramters checking in the deployment scripts
+process.env.TEST_DEPLOY_SCRIPTS = 'true';
```

Comment:
Might be worth adding some documentation about `TEST_DEPLOY_SCRIPTS` and `ZETO_NAME`, otherwise, the reader will need to parse it out from code reading.